src/Element.cc

```diff
@@ -697,22 +744,54 @@ void Element::PrintValues(const std::string &_prefix,
                           bool _includeDefaultElements,
                           bool _includeDefaultAttributes,
                           const PrintConfig &_config) const
+{
+  std::ostringstream ss;
+  sdf::Errors errors;
```

Comment:
nit: This could be implemented using `PrintValues` that takes an `_errors` instead of `PrintValuesImpl`.

Reply:
Done in https://github.com/gazebosim/sdformat/pull/1141/commits/83e7072e9da47fb738772bc13a998f1bca60928d.